src/libraries/System.Private.Xml/src/System/Xml/Core/XmlReader.cs

```diff
@@ -364,6 +373,7 @@ public virtual string ReadContentAsString()
             {
                 throw CreateReadContentAsException(nameof(ReadContentAsString));
             }
+
             return InternalReadContentAsString();
         }
 
```

Comment:
For below `public virtual object ReadContentAs(Type returnType, IXmlNamespaceResolver namespaceResolver)` `IXmlNamespaceResolver namespaceResolver` is nullable